src/System.DirectoryServices.AccountManagement/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -1,4 +1,23 @@
-<!-- Temporary workaround to get build to work. -->
-credui.dll!CredUIParseUserNameW
-advapi32.dll!LsaNtStatusToWinError
 activeds.dll!ADsOpenObject
+advapi32.dll!ImpersonateLoggedOnUser
+advapi32.dll!LogonUserW
+advapi32.dll!LsaClose
+advapi32.dll!LsaFreeMemory
+advapi32.dll!LsaLookupSids
+advapi32.dll!LsaNtStatusToWinError
+advapi32.dll!LsaOpenPolicy
```

Comment:
It is needed as it's not even onecore compatible yet -- however I trimmed this file further.